test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -967,7 +967,7 @@ def setup
       cr = FactoryBot.create(:compute_resource, :vmware, :uuid => 'Solutions',
```

Comment:
If you're changing the closing indent, you should also fix this line. Looks like there's a space too much

Reply:
I did change this manually now, but that would be the job of a (yet to be added) multi line cop.